walk-safe/src/Components/MapView/MapView.js

```diff
@@ -0,0 +1,34 @@
+import React, { Component } from 'react';
+import { Map, GoogleApiWrapper } from 'google-maps-react';
+
+const mapStyles = {
+  width: '40%',
+  height: '40%'
+};
+
+export class MapView extends Component {
+  render() {
+    return (
+      <div className='map-container'>
+        <Map
```

Comment:
Psyched about this map view - looking forward to hearing more about how it's structured!